daml-script/export/integration-tests/golden/BUILD.bazel

```diff
@@ -48,17 +48,6 @@ daml_test(
     deps = ["//daml-script/daml:daml-script.dar"],
 )
 
-daml_compile(
-    name = "interface-choices",
-    srcs = [
-        ":interface-choices/InterfaceChoices.daml",
-        ":interface-choices/InterfaceChoices/Asset.daml",
-        ":interface-choices/InterfaceChoices/Token.daml",
-    ],
-    dependencies = ["//daml-script/daml:daml-script-1.15.dar"],
-    target = "1.15",
-)
-
```

Comment:
how does the test below compile without this? Could we port it to 2.1 instead of removing it? or have we finally decided to kill `daml ledger export`?

Reply:
Good catch! I don't know why I deleted that, this is weird. My guess is that IJ or vscode was showing me that it was unused, and I thought it it was dead code. Which is [technically is](https://github.com/digital-asset/daml/blob/1971ccb9306206e6ddee035e00a3f2b479e70312/daml-script/export/integration-tests/golden/util.bzl#L37) at the moment.

Fixed.

Reply:
oh! good find, thanks for looking into it :)